www/source/partials/docs/_bp-planwriting-guide.html.md.erb

```diff
@@ -67,16 +67,16 @@ You can read more about [callbacks](/docs/reference/#reference-callbacks) here.
 
 #### Do's
 
-- `do_prepare()` is a good place to set environment variables and set the table to build the software. Think of it as a good place to do patches.
+- `do_prepare()` (`Invoke-Prepare` in a `plan.ps1`) is a good place to set environment variables and set the table to build the software. Think of it as a good place to do patches.
 
 #### Don't's
 
-- You should never call `exit` within a build phase. You should instead return an exit code such as `return 1` for failure, and `return 0` for success.  
-- If you clone a repo from git, you must override `do_verify()` to `return 0`.
+- You should never call `exit` within a build phase. In a `plan.sh`, you should instead return an exit code such as `return 1` for failure, and `return 0` for success. In a `plan.ps1` you should call `Write-Exception` or `throw` an exception upon failure.
+- If you clone a repo from git, you must override `do_verify()` to `return 0` or `Invoke-Verify` with an empty implementation.
```

Comment:
This is totally good, I'm just beginning to wonder if there isn't a good way to make these somehow togglable or easily switchable, or present them in a consistent way. Like for example, if I'm reading this and I don't know anything about Windows, I might not realize that `Invoke-Verify` doesn't apply to me here.) cc @ryankeairns

Reply:
hmm yeah, presuming by your comments that `Invoke-Verify` only applies to Windows, we may want to clearly delineate on platform specific instructions. For example, after the first sentence use list items or blockquotes to identify each platform.

You should never call `exit` within a build phase.
> **For a plan.sh** you should...
> **For a plan.ps1** you should...